specification/applicationinsights/data-plane/Monitor.Exporters/preview/2020-09-15/swagger.json

```diff
@@ -0,0 +1,1044 @@
+              },
+              "baseData": {
+                  "$ref": "#/definitions/Domain",
+                  "description": "The data payload for the telemetry request"
+              }
+          }
+      },
+      "DataPoint" : {
+        "x-ms-client-name": "MetricDataPoint",
+          "type": "object",
+          "description": "Metric data single measurement.",
+          "required": [
+              "name",
+              "value"
+          ],
+          "properties": {
+              "ns": {
+                  "x-ms-client-name": "namespace",
+                  "type": "string",
+                  "description": "Namespace of the metric.",
+                  "maxLength": 256
+              },
+              "name": {
+                  "type": "string",
+                  "description": "Name of the metric.",
+                  "maxLength": 1024
+              },
+              "kind": {
+                  "x-ms-client-name": "dataPointType",
+                  "$ref": "#/definitions/DataPointType",
+                  "description": "Metric type. Single measurement or the aggregated value."
+              },
+              "value": {
+                  "type": "number",
+                  "description": "Single value for measurement. Sum of individual measurements for the aggregation.",
+                  "format": "double"
+              },
+              "count": {
+                  "type":"number",
+                  "format": "int32",
+                  "description": "Metric weight of the aggregated metric. Should not be set for a measurement.",
+                  "x-nullable": true
+              },
+              "min": {
+                  "type":"number",
+                  "format": "double",
+                  "description": "Minimum value of the aggregated metric. Should not be set for a measurement.",
+                  "x-nullable": true
+              },
+              "max": {
+                  "type":"number",
+                  "format": "double",
+                  "description": "Maximum value of the aggregated metric. Should not be set for a measurement.",
+                  "x-nullable": true
+              },
+              "stdDev": {
+                  "type":"number",
+                  "format": "double",
+                  "description": "Standard deviation of the aggregated metric. Should not be set for a measurement.",
+                  "x-nullable": true
+              }
+          }
+      },
+      "DataPointType" : {
+          "type": "string",
+          "description": "Type of the metric data measurement.",
+          "enum": [
+              "Measurement",
+              "Aggregation"
+          ]
+      },
+      "Domain" : {
+          "x-ms-client-name": "MonitorDomain",
+          "type": "object",
+          "description": "The abstract common base of all domains.",
```

Comment:
this base class doesn't seem to actually contain anything useful? What's the point of it?

Reply:
It's required for Java and C# to define the input list of telemetry events.